app/src/main/java/dev/arkbuilders/rate/presentation/settings/AboutScreen.kt

```diff
@@ -50,275 +22,12 @@ fun AboutScreen(navigator: DestinationsNavigator) {
         },
     ) {
         Box(modifier = Modifier.padding(it)) {
-            Content(navigator)
-        }
-    }
-}
-
-@Composable
-private fun Content(navigator: DestinationsNavigator) {
-    val ctx = LocalContext.current
-
-    var btcDialogVisible by remember { mutableStateOf(false) }
-    var ethDialogVisible by remember { mutableStateOf(false) }
-
-    QRCryptoDialog(
-        visible = btcDialogVisible,
-        title = stringResource(R.string.about_donate_btc),
-        wallet = stringResource(R.string.about_btc_wallet),
-        fileName = "ArkQrBtc.jpg",
-        qrBitmap = R.drawable.qr_btc,
-    ) {
-        btcDialogVisible = false
-    }
-
-    QRCryptoDialog(
-        visible = ethDialogVisible,
-        title = stringResource(R.string.about_donate_eth),
-        wallet = stringResource(R.string.about_eth_wallet),
-        fileName = "ArkQrEth.jpg",
-        qrBitmap = R.drawable.qr_eth,
-    ) {
-        ethDialogVisible = false
-    }
-
-    Column(
-        modifier =
-            Modifier
-                .verticalScroll(rememberScrollState()),
-    ) {
-        Column(
-            modifier = Modifier.padding(horizontal = 16.dp),
-            horizontalAlignment = Alignment.CenterHorizontally,
-        ) {
-            Icon(
-                modifier = Modifier.padding(top = 32.dp),
-                painter = painterResource(id = R.drawable.ic_about_logo),
-                contentDescription = "",
-                tint = Color.Unspecified,
+            ArkAbout(
+                appName = stringResource(id = R.string.app_name),
```

Comment:
Oh, I forgot to add modifier arg)

Reply:
I remove unused resources in https://github.com/ARK-Builders/ARK-Rate/pull/108/commits/d7373dd2c3cecc6d3b259f0ace02874f54f6a316
I'll add modifier later